examples/ffd/ffd_cyberon.cmake

```diff
@@ -0,0 +1,196 @@
+)
+
+
+#**********************
+# Flags
+#**********************
+set(APP_COMPILER_FLAGS
+    -Os
+    -g
+    -report
+    -fxscope
+    -mcmodel=large
+    -Wno-xcore-fptrgroup
+    ${CMAKE_CURRENT_LIST_DIR}/src/config.xscope
+)
+
+set(APP_COMPILE_DEFINITIONS
+    configENABLE_DEBUG_PRINTF=1
+    PLATFORM_USES_TILE_0=1
+    PLATFORM_USES_TILE_1=1
+    QSPI_FLASH_FILESYSTEM_START_ADDRESS=${FILESYSTEM_START_ADDRESS}
+    QSPI_FLASH_MODEL_START_ADDRESS=${MODEL_START_ADDRESS}
+    QSPI_FLASH_CALIBRATION_ADDRESS=${CALIBRATION_PATTERN_START_ADDRESS}
+    ASR_CYBERON=1
+)
+
+set(APP_LINK_OPTIONS
+    -report
+    -lotp3
+    ${CMAKE_CURRENT_LIST_DIR}/src/config.xscope
+)
+
+set(APP_COMMON_LINK_LIBRARIES
+    sln_voice::app::ffd::ap
+    sln_voice::app::asr::Cyberon
+    sln_voice::app::ffd::xk_voice_l71
+)
+
+#**********************
+# Tile Targets
+#**********************
+set(TARGET_NAME tile0_example_ffd_cyberon)
+add_executable(${TARGET_NAME} EXCLUDE_FROM_ALL)
+target_sources(${TARGET_NAME} PUBLIC ${APP_SOURCES})
+target_include_directories(${TARGET_NAME} PUBLIC ${APP_INCLUDES} ${RTOS_CONF_INCLUDES})
+target_compile_definitions(${TARGET_NAME} PUBLIC ${APP_COMPILE_DEFINITIONS} THIS_XCORE_TILE=0)
+target_compile_options(${TARGET_NAME} PRIVATE ${APP_COMPILER_FLAGS})
+target_link_libraries(${TARGET_NAME} PUBLIC ${APP_COMMON_LINK_LIBRARIES})
+target_link_options(${TARGET_NAME} PRIVATE ${APP_LINK_OPTIONS})
+unset(TARGET_NAME)
+
+set(TARGET_NAME tile1_example_ffd_cyberon)
+add_executable(${TARGET_NAME} EXCLUDE_FROM_ALL)
+target_sources(${TARGET_NAME} PUBLIC ${APP_SOURCES})
+target_include_directories(${TARGET_NAME} PUBLIC ${APP_INCLUDES} ${RTOS_CONF_INCLUDES})
+target_compile_definitions(${TARGET_NAME} PUBLIC ${APP_COMPILE_DEFINITIONS} THIS_XCORE_TILE=1)
+target_compile_options(${TARGET_NAME} PRIVATE ${APP_COMPILER_FLAGS})
+target_link_libraries(${TARGET_NAME} PUBLIC ${APP_COMMON_LINK_LIBRARIES})
+target_link_options(${TARGET_NAME} PRIVATE ${APP_LINK_OPTIONS} )
+unset(TARGET_NAME)
+
+#**********************
+# Merge binaries
+#**********************
+merge_binaries(example_ffd_cyberon tile0_example_ffd_cyberon tile1_example_ffd_cyberon 1)
+
+#**********************
+# Create run and debug targets
+#**********************
+create_run_target(example_ffd_cyberon)
+create_debug_target(example_ffd_cyberon)
+
+#**********************
+# Create data partition support targets
+#**********************
+set(TARGET_NAME example_ffd_cyberon)
+set(DATA_PARTITION_FILE ${TARGET_NAME}_data_partition.bin)
+set(MODEL_FILE ${TARGET_NAME}_model.bin)
+set(FATFS_FILE ${TARGET_NAME}_fat.fs)
+set(FLASH_CAL_FILE ${LIB_QSPI_FAST_READ_ROOT_PATH}/lib_qspi_fast_read/calibration_pattern_nibble_swap.bin)
+
+add_custom_target(${MODEL_FILE} ALL
+    COMMAND ${CMAKE_COMMAND} -E copy ${CYBERON_COMMAND_NET_FILE} ${MODEL_FILE}
+    COMMENT
+        "Copy Cyberon NET file"
+    VERBATIM
+)
+
+create_filesystem_target(
+    #[[ Target ]]                   ${TARGET_NAME}
```

Comment:
Wondering why this is commented out?

Reply:
actually it is not commented out, they are bracket comments: https://cmake.org/cmake/help/latest/manual/cmake-language.7.html#bracket-comment